src/net.rs

```diff
@@ -493,7 +527,7 @@ impl<S: NetworkStream> NetworkStream for HttpsStream<S> {
 
 /// A Http Listener over SSL.
 #[derive(Clone)]
-pub struct HttpsListener<S: Ssl> {
+pub struct HttpsListener<S: SslServer> {
```

Comment:
Will this break constructing `HttpsListener` for users who don't have the `SslServer` imported?